flatbuffers/Quote.fbs

```diff
@@ -74,4 +74,8 @@ table Quote {
     bid: [QuoteEntry];
     /// Offer quote entries.
     offer: [QuoteEntry];
+    /// Dodd-Frank mid price.
+    mid_price: double;
+    /// Swaps only, far leg Dodd-Frank mid price
+    far_mid_price: double;
```

Comment:
Do we need to confirm if we'll have a far mid? It it's in swap points it'll be a single mid.

Reply:
Checked SCB and they provide mid per leg + mid in swap points. So it would be a single mid_price.

Reply:
We did raise that in the initial call last week, and the answer was yes, a mid price for each leg.
I also see backend are proceeding with mid price and far mid price
https://github.com/reactivemarkets/schema/pull/1159